web/src/main/java/no/nav/modiapersonoversikt/utils/ConcurrencyUtils.kt

```diff
@@ -45,10 +46,13 @@ object ConcurrencyUtils {
         val mdc = MDC.getCopyOfContextMap()
         val context = AuthContextUtils.getContext()
         val requestAttributes: RequestAttributes? = RequestContextHolder.getRequestAttributes()
+        val authheader = AuthHeaderCapture.header.get()
```

Comment:
Skjedd noe rart i rebasingen her tror jeg. For disse endringene er allerede i `dev`